### PR TITLE
Add support for bucket notifications

### PR DESCRIPTION
**Pull request by max-lobur**
## what
* Add support for bucket notifications
* Bypass all vars to log storage module

## why
* To enable log collection

### CODE REVIEW

**Review by mcalhoun — APPROVED** [review]
LGTM!

File: variables.tf [hunk 1]
```diff
@@ -81,3 +81,21 @@ variable "kms_policy_source_json" {
   default     = ""
   description = "Additional IAM policy document that can optionally be passed and merged with exported document"
 }
+
+variable "bucket_notifications_enabled" {
+  type        = bool
+  description = "Send notifications for the object created events. Used for 3rd-party log collection from a bucket"
```
Comment (korenyoni) [inline variables.tf:87]:
```suggestion
  description = "Send notifications for the object created events. Used for 3rd-party log collection from a bucket."
```

File: variables.tf [hunk 2]
```diff
@@ -81,3 +81,21 @@ variable "kms_policy_source_json" {
...
+
+variable "bucket_notifications_type" {
+  type        = string
+  description = "Type of the notification configuration. Only SQS is supported."
+  default     = "SQS"
+}
+
+variable "bucket_notifications_prefix" {
+  type        = string
+  description = "Prefix filter. Used to manage object notifications"
```
Comment (korenyoni) [inline variables.tf:99]:
```suggestion
  description = "Prefix filter. Used to manage object notifications."
```

**Review by korenyoni — APPROVED** [review]
LGTM with minor nitpicks.

**Pull request merged**